src/Microsoft.AspNetCore.Razor.Design/Internal/AssemblyTagHelperDescriptorResolver.cs

```diff
@@ -216,6 +234,27 @@ private IEnumerable<TagHelperDescriptor> GetFakeDescriptors(IEnumerable<TagHelpe
 
             return fakeDescriptors;
         }
+
+        private TagHelperDescriptor GetFakeDescriptorForTypeName(TagHelperDescriptor descriptor, string typeName)
```

Comment:
Ahhh, well in that case hide n seek. You set the TypeName 3 times, only need to do it once 😉